shell/config/product/explorer.js

```diff
@@ -192,7 +192,30 @@ export function init(store) {
   headers(WORKLOAD_TYPES.DAEMON_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', 'Current', 'Desired', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
   headers(WORKLOAD_TYPES.REPLICA_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', 'Current', 'Desired', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
   headers(WORKLOAD_TYPES.STATEFUL_SET, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Ready', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
-  headers(WORKLOAD_TYPES.JOB, [STATE, NAME_COL, NAMESPACE_COL, WORKLOAD_IMAGES, WORKLOAD_ENDPOINTS, 'Completions', 'Duration', POD_RESTARTS, AGE, WORKLOAD_HEALTH_SCALE]);
+  headers(WORKLOAD_TYPES.JOB, [
+    STATE,
+    NAME_COL,
+    NAMESPACE_COL,
+    WORKLOAD_IMAGES,
+    WORKLOAD_ENDPOINTS,
+    'Completions',
```

Comment:
this should be defined on `table-headers` and imported to `explorer.js` to be used as a constant, just to keep the same pattern of usage/definition

Reply:
Ignore this comment because of the getter stated here... Moving this to table-headers would imply that we would need state access there and that's not the goal. Thanks 🙏